src/Storage/AuthorizationCodeStorage.php

```diff
@@ -42,7 +43,20 @@ private function getUserIdByCode( $code ) {
 			return null;
 		}
 
-		return absint( $users[0]->ID );
+		if ( count( $users ) > 1 ) {
+			// This should never happen.
+			// If it does, something is wrong, so it's best to not return any user.
+			return null;
+		}
+
+		$user = $users[0];
```

Comment:
Could we use a `foreach` loop and return the user for whom the user_meta exists? That way we would make up for an inaccurate query and still return the right user instead of bailing out if the first one happens to be the wrong one.

Reply:
Good point! Any ideas as to how we highlight this to ourselves that such a thing has happened provided that we can't simply log it?

Reply:
Phpcs says:

> phpcs: error_log() found. Debug code should not normally be used in production.

Is there a "standard" way to log from plugins? Or should I silence phpcs here?

Reply:
Does it complain if you hide it behind a `if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {`?

Reply:
No standard way to log as such, we would have to silence phpcs

Reply:
> Does it complain if you hide it behind a if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {?

Assuming WP_DEBUG would be false in prod, won't we be forced to just silence it?

Reply:
Implemented